packages/router-component-store/package.json

```diff
@@ -10,7 +10,7 @@
     "rxjs": "^7.0.1"
   },
   "dependencies": {
-    "tslib": "^2.3.0"
+    "tslib": "2.3.1"
```

Comment:
issue: set up a package group with `"includePaths: ["packages/**/package.json"]`, using a different `rangeStrategy` so that semantic version ranges are preserved for package `dependencies`.